src/Component/FetchData.js

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect, useState } from 'react';
+
+function FetchData() {
+  const [fetchData, SetFechedData] = useState([]);
+  const [dataLoading, SetDataLoad] = useState(false);
+  const [errorLoad, SetErrorLoad] = useState(false);
+
+  // fetch data
+  useEffect(() => {
+    const loadData = async () => {
+      SetDataLoad(true);
+
+      try {
+        const fetchQuote = await fetch('https://api.api-ninjas.com/v1/quotes?limit=1', {
+          method: 'GET',
+          headers: { 'X-Api-Key': 'JBqUI+OJghdj6tMuxMCPkw==iytRIlFUQeiKleIN' },
+
+        });
+        const data = await fetchQuote.json();
+
+        SetFechedData(data);
+      } catch (error) {
+        SetErrorLoad(true);
+      }
+      SetDataLoad(false);
+    };
+    loadData();
+  }, [SetFechedData, SetErrorLoad, SetDataLoad]);
+
+  if (dataLoading) {
+    return <div className="loading">Loading...</div>;
+  }
+  if (errorLoad) {
+    return <div className="error">please try again after 5 hours</div>;
+  }
+  return (
+    <div>
+      { fetchData.map((data) => (
+        <p className="quote" key={data.quote}>
+          {data.quote}
+        </p>
+      ))}
+    </div>
+  );
+}
+
```

Comment:
- [OPTIONAL] It is more semantic to call this the `Quotes` component instead of the `FetchData` component. Remember you are displaying quotes from an endpoint. Naming a function `fetchData` should be for making `GET` requests to API endpoints and not a name of a component. Naming functions and components properly makes it easier to debug and know which function is responsible for certain actions on a webpage. It would be nice to implement this suggestion as I strongly recommend it. 😄 💻

Reply:
thank you so much @Gambit142  for this review I will make the changes as you suggested